### DISCUSSION TITLE
数据块转换为UTF-8编码时出现问题

### DISCUSSION THREAD

**Discussion by yblydhhj** [nextai-translator/nextai-translator#1243] [Q&A]
0.3.8版本出现如下图所示问题
<img width="1280" alt="屏幕截图 2024-01-10 211311" src="https://github.com/openai-translator/openai-translator/assets/156202880/6ec2b00a-4967-4717-ae81-b2c5592b3179">

**Comment by EdisonSean** [comment]
同问

**Comment by EdisonSean** [comment]
兄弟我是这样解决的：
1、卸载通过.msi安装的openai translator
2、下载OpenAI.Translator_0.3.8_x64-setup.exe，.exe结尾的
3、再卸载通过.exe安装的openai translator，卸载的时候会提示你是否删除data（我觉得问题关键在这个 删除数据），点yes。
4、再安装.msi的openai translator
5、管理员运行

不严谨，招不到问题，但能用了

**Reply by A1den23** [reply]
同样出现了这个问题，版本0.3.27，卸载重新安装仍然没解决
